### DISCUSSION TITLE
Probe Count

### DISCUSSION THREAD

**Discussion by Ackta** [Guilouz/Creality-Helper-Script-Wiki#434] [General]
Hi, i`m not able to change  from the macro, the count of probe,sameone explain me how?i`m new in klipper, and my bed is very concave.Thanks

**Comment by Foxtrotc7** [comment]
Just write 6 in the box and it probes 6,6 mesh. It is the maximum for lagrange algorithm.

**Reply by Ackta** [reply]
i can use bicubic algor with kamp?

**Reply by Foxtrotc7** [reply]
I'm not sure if it applies to KAMP, only bed leveling. I haven't tried to fiddle with the configurations for this, so you need to wait another to answer this.

**Comment by aDiii** [comment]
it is possible to make a bigger mesh using bicubic algorithm. but you also have to change the pr version of prtouch from 2 to 1.

[prtouch_v2]
pr_version: 1 #2

then you have to delete all the tri_min_hold_1 to the tri_min_hold_36 lines and same for tri_min_max_1-36.

then just adjust your bed mesh as you want

[bed_mesh]
algorithm: bicubic
probe_count: 8,8 #for example 8x8

**Reply by Ackta** [reply]
thanks

**Reply by KiLLie2k** [reply]
literally factory reset my printer trying to figure this out, first layer on the k1max is garbage.  not sure why this wouldn't be included in the docs as its so vague. being stuck to 6x6 lagrange is horrible.

**Reply by KiLLie2k** [reply]
my hero haha

**Reply by Derdness** [reply]
thanks

**Reply by LordQuerix** [reply]
thank you very much

**Reply by creichert** [reply]
I do notice when doing this that the screen (not guppy) does fail to work after some time. Does it work fully using Guppy w/ creality disabled?

**Comment by onkytonk** [comment]
> Hi, i`m not able to change from the macro, the count of probe,sameone explain me how?i`m new in klipper, and my bed is very concave.Thanks

I am having the same issue when trying to use the "BED_LEVELING" macro. 
All the other inputs will respond to whatever input I type, but probe_count always fallsback to whatever is in the printer.cfg.

I've tried:
7
7,7
7, 7
49
All with no success in changing the probe count on the fly. 

I've tried switching from lagrange to bicubic, but this has no affect.

Any ideas? Does this work for anybody?

**Reply by aDiii** [reply]
Did you try the method i mentioned?

**Reply by onkytonk** [reply]
I don’t have any issues changing the probe count if I change it in the printer.cfg. 

It’s that I’m wanting to be able to change it on the fly using a gcode or macro.

**Comment by christ1013** [comment]
bonjour, j'ai effectué le changement et tout était parfais mais erreur webcam, mais au redémarrage, plus de webcam et écran noir sur l'imprimante (mais elle fonctionne depuis web (fluiid et creality print) 😥

**Comment by argolomb** [comment]
After change pr_version from 2 to 1 and remove tri min and max, work until a until turn off, after turn on screen stuck on creality logo.

**Reply by creichert** [reply]
Did you figure this part out?

**Reply by argolomb** [reply]
No. I made a factory reset and leave with default probe count.

**Reply by aDiii** [reply]
Did you also change the algorithm to bicubic? Actually it shouldnt matter, it should just give you an error if not. Did you try connecting to mainsail or via ssh while it was stuck? Maybe any errors?

**Reply by creichert** [reply]
I tried your method yesterday and it worked with no issue. Using bicubic to chase mesh issues and warping on my bed to get a flawless first layer.

It slightly improved my first layer on the Max, but I was really hoping to get closer to flawless. I believe local warping spots on the aluminum bed between mesh points cannot be compensated.

I don't mean to change the issue, but if you have any advice after looking at these that would help. I have tried literally everything and think it's just a probe/load cells inaccuracy. Changing my z_offset tends to make one spot better and others worse. Same w/ flow, that's pretty much dialed in.

<img width="1017" height="1028" alt="image" src="https://github.com/user-attachments/assets/fd565364-56f2-48ad-8e30-6af627ccf6b6" />
<img width="1294" height="1193" alt="image" src="https://github.com/user-attachments/assets/0b241904-7023-40a4-97a1-074aba7be29e" />

Here's a picture of my mesh: <img width="995" height="722" alt="image" src="https://github.com/user-attachments/assets/ffa93b88-646b-4d9f-96fe-bbca5d54dd52" />

In comparison, an Ender 3 w/ a BTT Eddy puts down a flawless first layer no matter what I do! So I'm a bit frustrated w/ my Max at this point and I think getting a Cartographer is a challenge using the DXC extruder.

**Reply by christ1013** [reply]
Bonjour, pas de problème de webcam ou d'écran noir ?

**Reply by christ1013** [reply]
passe en 11x11 voir plus, c'est plus long mais plus précis pour les corrections
